dogen/generator.py

```diff
@@ -175,18 +175,6 @@ def _handle_additional_scripts(self):
                 self.log.debug("Copying '%s' file to target scripts directory..." % f)
                 shutil.copy(f, output_scripts)
 
-    def _handle_custom_repo_files(self):
-        self.cfg['additional_repos'] = []
-        repo_files = glob.glob(os.path.join(self.output, "scripts", "*.repo"))
-
-        if not repo_files:
```

Comment:
i think you can make it more readable with something like:

for f in sorted(glob.glob(os.path.join(self.output, "scripts", "*.repo")):
  self.cfg .....
else:
  return